homeassistant/components/sensor/openevse.py

```diff
@@ -22,7 +22,7 @@
 SENSOR_TYPES = {
     'status': ['Charging Status', None],
     'charge_time': ['Charge Time Elapsed', 'minutes'],
-    'ambient_temp': ['Ambient Termperature', TEMP_CELSIUS],
+    'ambient_temp': ['Ambient Temperature', TEMP_CELSIUS],
```

Comment:
AFAIK this would change the `friendly_name` of certain openevse devices and thus also their `entity_id` which could break configurations.